pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -114,8 +118,9 @@ func (admitter *VMICreateAdmitter) Admit(_ context.Context, ar *admissionv1.Admi
 		causes = append(causes, deprecation.ValidateFeatureGates(devCfg.FeatureGates, &vmi.Spec)...)
 	}
 
-	netValidator := netadmitter.NewValidator(k8sfield.NewPath("spec"), &vmi.Spec, admitter.ClusterConfig)
-	causes = append(causes, netValidator.ValidateCreation()...)
+	for _, validateSpec := range admitter.SpecValidators {
+		causes = append(causes, validateSpec(k8sfield.NewPath("spec"), &vmi.Spec, admitter.ClusterConfig)...)
+	}
 
 	causes = append(causes, ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("spec"), &vmi.Spec, admitter.ClusterConfig)...)
```

Comment:
Please consider stating in the commit message that there is another instantiation of the network validator in `ValidateVirtualMachineInstanceSpec` that will be handled in a follow-up, so future reviewers could understand why following this change `pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go` is still dependent on `kubevirt.io/kubevirt/pkg/network/admitter`.

Reply:
Done

Reply:
Thank you for adding that info.
> ...  will be done in follow-up commit.

Make it sound like it will be a part of this PR. Please consider replacing "commit" with "PR".

Reply:
I fined it odd to mention PR in a commit message, "follow-up" can refer to another PR.
There is enough information to understand this change, and eventually reach to the PR description that has all the details in one place.

Reply:
I use "change" to avoid naming it a commit or PR.